README.md

```diff
@@ -1,4 +1,191 @@
-Astarte Dashboard
-=================
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
```

Comment:
I don't like all this HTML code here, we are trying to keep our markdown as close as possible to plain text.